README.md

```diff
@@ -20,6 +20,29 @@ Start from docker image (TODO):
 docker run -it -p 27999-28001:27999-28001 open-privacy/opv
 ```
 
+Create a new grant token by calling the control plane:
+```sh
+curl -X POST 'http://localhost:27999/api/v1/grants' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+        "allowed_actions": ["*"],
```

Comment:
I changed to `"allowed_http_methods"` to better reflect what it is, and I think it's better for us to reserve `allowed_actions` for any future definition of `actions`.